en/docs/setup/changing-to-mysql.md

```diff
@@ -98,6 +98,18 @@ After setting up the MySQL database. You can point the `WSO2_IDENTITY_DB` or
                       
            - `<IS-HOME>/dbscripts/mysql.sql`
            
+    !!! note     
+            Instead of defining `hostname`, `port`, `name` separately, you can define `url`
```

Comment:
the `hostname`,  `port`, and `name` separately, you can define the `url`

Reply:
Changed